zip-0226.rst

```diff
@@ -194,128 +197,146 @@ As in the Orchard protocol, the binding signature verification key, :math:`\math
-| Bytes           | Name        | Data Type                         | Description             |
-+=================+=============+===================================+=========================+
-| varies          | nAssetBurn  | compactSize                       | number of Assets burnt  |
-+-----------------+-------------+-----------------------------------+-------------------------+
-| 40*nAssetBurn   | vAssetBurn  | bytes[40][nAssetBurn]             | 32 bytes Asset type_t,  |
-|                 |             |                                   | 8 bytes of valueBalance |
-+-----------------+-------------+-----------------------------------+-------------------------+
++-------+-----------+-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
+| Bytes | Name      | Data Type                   | Description                                                                                                               |
++=======+===========+=============================+===========================================================================================================================+
+| 32    | AssetBase | byte[32]                    | For the Orchard-based ZSA protocol, this is the encoding of the Asset Base :math:`\mathsf{AssetBase}^{\mathsf{Orchard}}`. |
++-------+-----------+-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
+| 8     | valueBurn | :math:`\{1 .. 2^{64} - 1\}` | The amount of the AssetType being burnt.                                                                                  |
++-------+-----------+-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
 
-In terms of the Action size, the ZSA action size differs from the Orchard action size by 32 bytes (due to the addition of the :math:`\mathsf{AssetId}`). This implies that the size goes from 820 bytes in the Orchard action to 852 bytes in the ZSA Action.
 
 Security and Privacy Considerations
 ===================================
 
-- Even if the Orchard protocol and ZSA protocol do not share the same anonymity pool (nodes can keep track of the notes that where published with different transaction structures), the migration from one to the other is done automatically and seamlessly. The Orchard bundle will be replaced by the ZSA bundle and all ZEC notes will be fully spendable with the new transaction structure.
+- The notes generated via the ZSA protocol are different from those generated via the Orchard protocol. It will be possible to distinguish between notes generated via these two protocols. However, all ZEC notes will be fully spendable with the ZSA protocol transaction structure due to the built-in backward compatibility.
```

Comment:
As with any protocol upgrade, it will be possible to distinguish between notes generated by each protocol.

Reply:
Done.